backend/routes/indUsers.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express')
+const {
+    createIndUser,
+    getAllIndUsers
```

Comment:
line 4: getIndUserByID

Reply:
We left out that function as we weren't sure we'd need it for any functionality for v2 - I've added a ticket so we can discuss on Mon